pgatk/commands/ncbi_downloader.py

```
@@ -8,29 +12,92 @@
+    build = "GRCh37" if grch37 else "GRCh38"
+    prefix = f"{build}_latest_genomic"
+    genome_fna = os.path.join(output_dir, f"{prefix}.fna.gz")
+    gff_file = os.path.join(output_dir, f"{prefix}.gff.gz")
+
+    if generate_transcripts:
+        output_transcripts = os.path.join(output_dir, "transcripts.fa")
+        try:
+            click.echo(f"Running gffread to generate transcripts.fa ({build}) ...")
+            NcbiDataDownloader.generate_transcripts(genome_fna, gff_file, output_transcripts)
+            click.echo(f"Transcripts written to {output_transcripts}")
+        except (FileNotFoundError, Exception) as exc:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)